services/director-v2/src/simcore_service_director_v2/utils/dask.py

```diff
@@ -300,6 +304,32 @@ async def compute_service_log_file_upload_link(
     return value_link
 
 
+async def get_service_log_file_download_link(
+    user_id: UserID,
+    project_id: ProjectID,
+    node_id: NodeID,
+    file_link_type: FileLinkType,
+) -> Optional[AnyUrl]:
+    """Returns None if log file is not available (e.g. when tasks is not done)
+
+    : raises StorageServerIssue
+    : raises NodeportsException
+    """
+    try:
+        value_link = await port_utils.get_download_link_from_storage_v2(
```

Comment:
question: why do you call it value_link?

Reply:
By analogy with all the similar functions in the module (see e.g. ``compute_service_log_file_upload_link``) .